### PR TITLE
Docs: Update the React documentation to include changes to the `<OpenFormatProvider/>`

### PR DESCRIPTION
**Pull request by Sam-Archie**
**What**:
Update to the documentation to display the requirement of a network option that needs to be passed to the provider for it to function as expected.

**Why**:
If the provider is not given a network, the provider, and therefore the SDK along with all of its functions will not work.

**How**:

Update to the docs to show this to anyone using it should avoid any confusion.
**Checklist**:

- [x] Documentation
- [ ] Tests "N/A"
- [ ] Code complete "N/A"
- [ ] Changeset  "N/A"

### Images
Screenshot 2022-08-03 at 16 06 40

### CODE REVIEW

File: sdks/react/README.md [hunk 1]
```diff
@@ -22,20 +22,26 @@ You can [checkout an example Next.js to see how it works](https://github.com/sim
...
-      <OpenFormatProvider>{/* the rest of your app... */}</OpenFormatProvider>
+      <OpenFormatProvider
+        config={{ network: "https://matic-mumbai.chainstacklabs.com" }}
+      ></OpenFormatProvider>
     </>
   );
 }
 ```
 
+````
```
Comment (craigcoles) [inline sdks/react/README.md:43]: ✂️  This needs removing

File: sdks/react/README.md [hunk 2]
```diff
@@ -48,13 +54,13 @@ function MyComponent() {
 
   return <>{data && <pre>{JSON.stringify(data, null, 2)}</pre>}</>;
 }
-```
+````
```
Comment (craigcoles) [inline sdks/react/README.md:57]: Extra ` needs removing here

File: sdks/react/README.md [hunk 3]
```diff
@@ -22,20 +22,26 @@ You can [checkout an example Next.js to see how it works](https://github.com/sim
...
-import { OpenFormatProvider } from '@simpleweb/open-format-react';
+import { OpenFormatProvider } from "@simpleweb/open-format-react";
 
 function App() {
   return (
     <>
-      <OpenFormatProvider>{/* the rest of your app... */}</OpenFormatProvider>
+      <OpenFormatProvider
+        config={{ network: "https://matic-mumbai.chainstacklabs.com" }}
+      ></OpenFormatProvider>
```
Comment (craigcoles) [inline sdks/react/README.md:37]:
```suggestion
      <OpenFormatProvider
        config={{ network: "https://matic-mumbai.chainstacklabs.com" }}
      >
        {/* the rest of your app... */}
      </OpenFormatProvider>
```

File: sdks/react/README.md [hunk 4]
```diff
@@ -22,20 +22,26 @@ You can [checkout an example Next.js to see how it works](https://github.com/sim
 
 ## Getting started
 
-Start by adding the `OpenFormatProvider` around your app.
+- Start by adding the `OpenFormatProvider` around your app.
+- The provider accepts a config object where you set your network url.
+- We have provided a hard coded example of a test network on mumbai. A full list of networks can found [here.](https://chainlist.org/)
```
Comment (craigcoles) [inline sdks/react/README.md:27]:
```suggestion
- We have provided an example using a test network on Mumbai. A list of networks can found on [chainlist.org](https://chainlist.org/).
```

File: sdks/react/README.md [hunk 5]
```diff
@@ -22,20 +22,26 @@ You can [checkout an example Next.js to see how it works](https://github.com/sim
 
 ## Getting started
 
-Start by adding the `OpenFormatProvider` around your app.
+- Start by adding the `OpenFormatProvider` around your app.
+- The provider accepts a config object where you set your network url.
```
Comment (craigcoles) [inline sdks/react/README.md:26]:
```suggestion
- The provider accepts a config object where you set the network. This must be a full URL.
```

**Review by craigcoles — CHANGES_REQUESTED** [review]
There is a few changes to tighten this all up.

It looks like your prettier is also changing all the quotes to single quotes. There isn't a problem with this, but would be good to just make sure all the examples are consistent.

File: sdks/react/README.md (hunk 3 above)
Comment (Sam-Archie) [reply sdks/react/README.md:37]: ✅

**Review by KateHoward10 — APPROVED** [review]

**Review by craigcoles — APPROVED** [review]

**Pull request merged**